src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/smart-list/fixtures/smart_data_version_1_0_nvme_response.json

```diff
@@ -79,7 +79,7 @@
         "Local Time is:                      Thu Oct 24 10:17:06 2019 CEST",
         "Firmware Updates (0x16):            3 Slots, no Reset required",
         "Optional Admin Commands (0x0017):   Security Format Frmw_DL Self_Test",
-        "Optional NVM Commands (0x005f):     Comp Wr_Unc DS_Mngmt Wr_Zero Sav/Sel_Feat Timestmp",
+        "Optional NVM Commands (0x005f):     Comp Wr_Unc DS_Mngmt Wr_Zero Sav/Sel_Feat Timestamp",
```

Comment:
It's possible (likely?) this is raw captured data and shouldn't be changed.

If files come from elsewhere / shouldn't be changed, I can add them to an excludes file so that future runs should ignore them.

Reply:
Yeah, probably the source data already had this typo... We could skip this one, but either way I don't think this is going to have any impact.